### DISCUSSION TITLE
Move existing erasure code server to distributed mode

### DISCUSSION THREAD

**Discussion by bollerdominik** [minio/minio#12114] [Q&A] [answered]
I have an existing server started in erasure code mode like this

`minio server data{1...10}`

The 10 disks are almost full and I would like to switch to an distributed mode. 

I tried to stop the server and restart it like this

`minio server http://host{1...2}.domain.com:9000/data{1...10}`

but I keep getting an error such as

>  disk is already being used in another erasure deployment.

How can I move to a distributed mode without reformatting the disks? It would be devastating if we are stuck with a 100TB node that we are unable to extend.

**Comment by klauspost** [comment, marked as answer]
You need to add another server pool since you cannot change the configuration of your existing one.

For your setup that would be...

`minio server http://host1.domain.com:9000/data{1...10} http://host2.domain.com:9000/data{1...10}`

**Reply by bollerdominik** [reply]
Thank you for your help. Starting the existing server works and I get the expected message:

> Error: Marking http://host2.domain.com:9000/minio/storage/data3/v30 temporary offline; caused by Post " http://host2.domain.com:9000/minio/storage/mnt/data/v30/readall?disk-id=&file-path=format.json&volume=.minio.sys": dial tcp xxxxxxxx:9000: connect: connection refused (*fmt.wrapError)
>        6: cmd/rest/client.go:138:rest.(*Client).Call()
>        5: cmd/storage-rest-client.go:151:cmd.(*storageRESTClient).call()
>        4: cmd/storage-rest-client.go:485:cmd.(*storageRESTClient).ReadAll()
>        3: cmd/format-erasure.go:405:cmd.loadFormatErasure()
>        2: cmd/format-erasure.go:325:cmd.loadFormatErasureAll.func1()
>        1: pkg/sync/errgroup/errgroup.go:122:errgroup.(*Group).Go.func1()
> Waiting for a minimum of x disks to come online (elapsed 5s)
> 


I then start the second server with the same command:
`minio server http://host1.domain.com:9000/data{1...10} http://host2.domain.com:9000/data{1...10}`

I then get following error log on the first server:

> ERROR Unable to initialize backend: All serverPools should have same deployment ID expected d686f1db-5e28-4c13-a3fa-474b0f0aadae, got e0bb83b5-6139-42a3-8c4b-38707792c43c

Unfortunately the deployment fails with this error.

//Edit: I think the issue is that my first server is called host2.domain.com and the second one was host1.domain.com. I changed the name and now it works. Thanks for your help!

**Reply by harshavardhana** [reply]
You can't mix two different deployments @bollerdominik

**Reply by DangNguyenTH4** [reply]
So what command should use in the second server, I also got this problem
